packages/dataviews/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Add `enableMoving` option to the `table` layout to allow or disallow column moving left and right. [#71120](https://github.com/WordPress/gutenberg/pull/71120)
 - Add infinite scroll support across all layout types (grid, list, table). Enable infinite scroll by providing an `infiniteScrollHandler` function in the `paginationInfo` prop and toggling the feature in the view configuration. ([#70955](https://github.com/WordPress/gutenberg/pull/70955))
 - Add support for modal in DataForm panel layouts. [#71212](https://github.com/WordPress/gutenberg/pull/71212)
+- Add support for ValidatedArrayControl [#71350](https://github.com/WordPress/gutenberg/pull/71350)
```

Comment:
This need to be moved to `packages/COMPONENTS/changelog`.

Reply:
Thanks! I moved it here c948fb1b51631f2b924de2f1ac0e9d8a92ca78d7